Traceback (most recent call last):
  File "test_detector.py", line 24, in <module>
    main()
  File "test_detector.py", line 11, in main
    cv2.imshow(str(client.pid), img3)
NameError: name 'img3' is not defined
